wallet/network/network.go

```diff
@@ -10,7 +10,7 @@ import (
 var ErrNetworkDoesNotHaveGRPCHostConfigured = errors.New("network configuration does not have any gRPC host set")
 
 type Network struct {
-	Name        string              `json:"name"`
+	Name        string              `json:"name" toml:"-"`
```

Comment:
I would prefer we use two dedicated struct instead of relying on the build tags to determine whether or not we should include the field. 

I am ok with reusing domain object for serialization when there is no differences, but ass soon as we start diverging, or introducing different version, it's better to back down and introduce dedicated structs, and convert them on the "IO" layer.

You can reuse most of the other structs, but the root struct should be different.

```go
type networkFileContent struct {
    LogLevel    vgencoding.LogLevel `json:"level"`
	TokenExpiry vgencoding.Duration `json:"tokenExpiry"`
	Port        int                 `json:"port"`
    //...
}

type Network struct {
    Name        string `json:"name"
    LogLevel    vgencoding.LogLevel `json:"level"`
	TokenExpiry vgencoding.Duration `json:"tokenExpiry"`
	Port        int                 `json:"port"`
    //...
}